src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -797,3 +798,20 @@ export const validateIfHasEvidence = (
 
 export const title10DatesRequired = formData =>
   get('view:isTitle10Activated', formData, false);
+
+export const ForwardingAddressViewField = ({ formData }) => {
+  const { effectiveDate } = formData;
+  return (
+    <div>
+      <EffectiveDateViewField formData={effectiveDate} />
+      <AddressViewField formData={formData} />
```

Comment:
@goldenmeanie Should the address in the screenshots be displayed differently? There's something that seems not quite right about them, but I can't put my finger on it. Maybe indent the address?

Reply:
It looks like the lines are separate paragraphs instead of just linebreaks

Reply:
We do also have an indent-and-highlight address style that is inconsistently applied ;)

Reply:
Yeah, this is much better. Could probably be its own PR, but I already did it, soo...
![screen shot 2018-10-12 at 12 36 06 pm](https://user-images.githubusercontent.com/24251447/46885158-c7b90e00-ce1c-11e8-826d-963c4b954872.png)